sigle/src/modules/layout/components/AppHeader.tsx

```diff
@@ -187,12 +193,29 @@ export const AppHeader = () => {
                 <StatusDot />
                 <Text>{user.username}</Text>
               </Flex>
-            </HoverCardTrigger>
-            <HoverCardContent onClick={handleLogout} sideOffset={16}>
-              logout
-              <HoverCardArrow />
-            </HoverCardContent>
-          </HoverCard>
+            </DropdownMenuTrigger>
+            <DropdownMenuContent sideOffset={8}>
+              <DropdownMenuItem>
+                <Link href={`/${user.username}`} passHref>
+                  <a>My blog</a>
+                </Link>
+              </DropdownMenuItem>
+              <DropdownMenuItem>
+                <Link href="/" passHref>
+                  <a>Dashboard</a>
+                </Link>
+              </DropdownMenuItem>
+              <DropdownMenuItem>
+                <Link href="/settings" passHref>
+                  <a>Settings</a>
+                </Link>
+              </DropdownMenuItem>
+              <DropdownMenuSeparator />
+              <DropdownMenuItem color="red" onClick={handleLogout}>
```

Comment:
@gregogun I think the theme switch under the flag is missing :)

Reply:
ah I thought this was to be added after the dark mode is fully integrated but then remembered we have the experimental flag! Will add it now :) 

Should I remove the existing one theme toggle button, or should it be left? 
![Screenshot 2022-04-19 at 08 27 35](https://user-images.githubusercontent.com/65421744/163949120-1f1c51c9-9743-4fce-b778-22fd76086427.png)

Reply:
You can remove it 👍